doc/tutorials/mediasdk-tutorials-cmd-reference.md

```diff
@@ -153,16 +180,34 @@ Tutorial demonstrates how to encode given YUV video stream (input-file) in H.264
 | input-file | Optional argument. Sets incoming input file containing uncompressed video. If file will not be specified, tutorial will work in the performance mode: input will be simulated by producing empty input frames filled with some color |
 | output-file | Optional argument. Sets raw H.264 video file to write output data. If file will not be specified, tutorial will work in the performance mode: it will process data, but will not produce any output. |
 
+## simple_3_encode
+**simple_encode *[-sw | -hw | -auto] -g WxH -b bitrate -f framerate [ input-file ] [ output-file ]***
+Tutorial demonstrates how to encode given p010 YUV video stream (input-file) in H.265 format. If output-file was specified, encoded bitstream will be written into it. If the input-file and output-file are omitted, tutorial can be used to estimate constructed pipeline’s performance (to some degree). Encoder is configured to receive input  data from system memory.
+
+| Option | Description |
+| ------:| -----------:|
+| -sw | Loads SW Media SDK Library implementation |
+| -hw | Loads HW Media SDK Library implementation |
+| -auto | Automatically choses Media SDK library implementation |
+| -g WxH | Mandatory option. Sets input video geometry, i.e. width and height. Example: -g 1920x1080. |
+| -b bitrate | Mandatory option. Sets bitrate with which data should be encoded, in KBits-per-second. Example: -b 5000 to encode data at 5Mbit. |
+| -f framerate | Mandatory option. Sets framerate with which data should be encoded in the form -f nominator/denominator. Example: -f 30/1. |
+
+| Argument | Description |
+| ------:| -----------:|
+| input-file | Optional argument. Sets incoming input file containing uncompressed video. If file will not be specified, tutorial will work in the performance mode: input will be simulated by producing empty input frames filled with some color |
```

Comment:
If file will not be specified -> If file is not be specified

Reply:
done